src/l500/l500-private.h

```diff
@@ -339,6 +340,28 @@ namespace librealsense
             hw_monitor & _hwm;
         };
 
+        /*  For RS2_OPTION_HARDWARE_SYNC_ENABLED
+            Not an advanced control: always off after camera startup (reset).
+            When enabled, the freefall control should turn off.
+        */
+        class hw_sync_option : public bool_option
+        {
+        public:
+            hw_sync_option( hw_monitor& hwm, std::shared_ptr< freefall_option > freefall_opt );
+
+            virtual void set( float value ) override;
+            virtual const char* get_description() const override
+            {
+                return "When enabled (disabled on startup), interrupts are handled according to GPIO10 level; shuts off the fall detect mechanism";
```

Comment:
Maybe we need to rephrase for the user's experience perspective

Reply:
Any suggestions? :)

Reply:
Even the other option (`RS2_OPTION_INTER_CAM_SYNC_MODE`) test "`Inter-camera synchronization mode: 0:Default, 1:Master, 2:Slave`" is not useful.

"`Enable multi-camera hardware synchronization mode; not compatible with free-fall detection`"?